README.md

```diff
@@ -56,44 +89,27 @@ tabling or plotting scripts.
 
 ### Generating tables from Krun results files
 
-There are two scripts used for generating tables,
-`bin/table_classification_summaries_main` and
-`bin/table_classification_summaries_others`. The `_main` script is intended to
-be used with the results of our full "warmup" experiment where a number of
-different VMs were tested on a number of different benchmarks, and the `_others`
-script is intended to be used with small numbers of VMs (1 or 2) and a set of
-benchmarks (e.g. Da Capo, Octane, etc.).
-
-Both script generate LaTeX code which needs to be translated to PDF (e.g. with
-`pdflatex`). The largest difference between the two scripts is that `_main`
-produces tables where every row is a VM (and benchmarks are grouped in blocks),
-and `_other` produces tables where every row is a benchmark (and benchmarks may
-be grouped by VMs, if more than one VM has been benchmarked).
-
-The two scripts are invoked slightly differently:
+`bin/table_classification_summaries_others` plots summary tables with various
```

Comment:
Users who started with `.csv` input should use `warmup_stats` to produce tables and plots, users who started with Krun output should use the `table_` script.